css-positioning/absolute.css

```diff
@@ -0,0 +1,39 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+
+  /* your code here */
```

Comment:
Don't see any of your code here 🤔 . Is there a positioning type that makes `absolute` easier to control? It would probably be beneficial to make use of it here, so positioning things in the `meme` class becomes more maintainable.